src/admin-views/editor/fieldset/settings-provider.php

```diff
@@ -72,7 +70,7 @@ class="tribe-ticket-editor-field-default_provider settings_field ticket_field"
 						 *
 						 * @param string $module - the name of the module
 						 */
-						echo apply_filters( 'tribe_events_tickets_module_name', esc_html( $module ) );
+						esc_html_e( apply_filters( 'tribe_events_tickets_module_name', $active_provider['name'] ) );
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "apply_filters( 'tribe_events_tickets_module_name', $active_provider['name'] )".

<sub>hash: 93d182b663c2f10fc1d9c799f64c65f5</sub>

Reply:
done